webpack.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  entry: './src/index.ts',
+  output: {
+    filename: './bundle.js',
+  },
+
+  // Enable sourcemaps for debugging webpack's output.
+  devtool: 'source-map',
```

Comment:
will this create source maps for production builds? as in will this bloat user's worker scripts?